plugins/services/src/js/utils/CreateServiceModalFormUtil.js

```diff
@@ -34,6 +34,119 @@ const CreateServiceModalFormUtil = {
   },
 
   /**
+   * Patch an object with an object
+   *
+   * This function:
+   * - Removes items whose patch value is `empty` (null, undefined, 0, or '')
```

Comment:
Just to be clear. If I had `gpus: 5` and go to `gpus: 0` how it will behave? 
Because Marathon will keep `gpus: 5` in case an update simply won't have that field.